File: VehiculeApp/app/src/main/java/com/example/vehiculeapp/api/ApiService.java
package com.example.vehiculeapp.api;

import com.example.vehiculeapp.model.ApiResponse;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {
    
    @GET("vehicle/list?icon=1&lang=en-gb&forceStandard=1&outputFormat=json&appToken=8qohg5a9c6q6x58szpyxizvp91yary3setxdxutl10dugtel1syjs6gmrp33oo40a356j2cxt6vdcpzg095drsym5blnyen0hi4bdq32j61clfux2i9vtuhr")
    Call<ApiResponse> getListVehicle();

}

File: VehiculeApp/app/src/main/java/com/example/vehiculeapp/api/ApiService.kt
package com.example.vehiculeapp.api

import com.example.vehiculeapp.model.ApiResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiService {

    @GET("vehicle/list")
    fun listVehicle(
        @Query("icon") icon: Int,
        @Query("lang") lang: String,
        @Query("forceStandard") forceStandard: Int,
        @Query("outputFormat") outputFormat: String,
        @Query("appToken") appToken: String
    ): Call<ApiResponse>
}